worker-workflow/src/main/resources/workflow-control.js

```diff
@@ -29,6 +29,21 @@ function onAfterProcessTask(eventObj) {
     processDocument(eventObj.rootDocument);
```

Comment:
It's not part of this ticket but renaming processDocument to routeDocument might help with the understanding.